src/routes/projects.rs

```diff
@@ -893,11 +893,13 @@ pub async fn project_edit(
                 }
 
                 {
```

Comment:
Wouldn't it be better to do the check here in code instead of in the query? We can bypass this entirely if the new slug is the same as the old slug.

Reply:
It may just be my unfamiliarity with the code, though I didn't see anything in that scope that would allow me to check the _current_ slug of the project without an additional query.

Reply:
`project_item` is the current project